### PR TITLE
 UNETR PyTorch Lightning Tutorial

### LINKED ISSUE #300: Add UNETR PyTorch Lightning Tutorial
**Is your feature request related to a problem? Please describe.**
Add tutorial for UNETR for BTCV multi-organ segmentation that supports the usage of PyTorch Lightning. 
**Describe the solution you'd like**
A new PR that achieves this. Note that it should support latest version of MONAI (0.6.0) and PyTorch Lightning (1.4.0).

### PR DESCRIPTION
**Pull request by ahatamiz**
Fixes #300 

### Description
This PR adds UNETR PyTorch Lightning Tutorial for multi-organ segmentation on BTCV dataset. 
### Status
**Ready**

### Checks

- [x] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

**Review by Nic-Ma — APPROVED** [review]
Hi @ahatamiz ,

Thanks for the enhanced tutorial for Lightning framework.
Could you please help keep the plot of training curve and validation images, etc.? I think we only need to remove the training log.
The loss curve and validation images can be references for users.

Thanks.

**Comment by Nic-Ma** [comment]
Thanks for the quick update!
I think it looks good to me now.
@wyli do you have any other comments?

Thanks.

**Comment by ahatamiz** [comment]
Hi @Nic-Ma 

Thanks for your review and comments. I added the training curves, visualizations etc.

**Comment by Nic-Ma** [comment]
Hi @ahatamiz ,

Could you please help add your 2 UNETR tutorials into the README doc:
https://github.com/Project-MONAI/tutorials/blob/master/README.md

Thanks in advance.

**Comment by ahatamiz** [comment]
> Hi @ahatamiz ,
> 
> Could you please help add your 2 UNETR tutorials into the README doc:
> https://github.com/Project-MONAI/tutorials/blob/master/README.md
> 
> Thanks in advance.

Hi @Nic-Ma,

Sure. Thanks for your reminder. I have updated the README doc to reflect the additions of these notebook tutorials. 

Thanks,
